### DISCUSSION TITLE
DietPi-Software | Tdarr

### DISCUSSION THREAD

**Discussion by TheTrueZeroTwo** [MichaIng/DietPi#6034] [Software Request]
#### Formal software information
- Software title | Tdarr
- Short description | conditional based transcoding application for automating media library transcode/remux
- Official URL (if available) | https://tdarr.io/
- GitHub URL (if available) | https://github.com/HaveAGitGat/Tdarr
- (Official) install documentation |https://docs.tdarr.io/docs/welcome/what/

#### Are there similar/alternative software titles available with DietPi-Software?

- ...

#### What makes your requested software better than the above solutions, if available?
N/A
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
According to the official install instructions there are post installation steps to complete. you have to make a service for the node and or the server. you also have to change config files if the node and server are not on the same computer. Allows installation as well to less experienced Linux users.
- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. Run install script to update/install server and node
2. change config file if using other computer as a node
3. change path translations to point to where media is kept
4. create service to have server and/or node always running

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
Yes, I could help out with this but i am still learning the best practices on Linux coding
- ...

**Comment by MichaIng** [comment]
It would be great if you could full the software request template to not leave all research to us what Tdarr actually is etc 😉.

**Comment by TheTrueZeroTwo** [comment]
sorry left the < !-- --> in

**Comment by MichaIng** [comment]
Ah okay, thanks 🙂.

**Comment by MichaIng** [comment]
Converted to a discussion. This can be now voted for via arrow button at OP.

**Comment by superturrican** [comment]
This would be very useful to have! Being able to run transcoding jobs automatically without effort would be ideal.
